decidim-dev/lib/decidim/dev/test/rspec_support/i18n.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.configure do |config|
   config.before(:suite) do
     I18n.config.enforce_available_locales = false
-    I18n.available_locales = %w(en ca es)
     Decidim.available_locales = %w(en ca es)
+    I18n.available_locales = Decidim.available_locales
   end
 end
```

Comment:
Since now the generated app locales match this configuration, I'd say we don't. I'd kill this file and see what happens.